<?xml version="1.0" encoding="utf-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Régulation de Température du Four</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <widget class="QLabel" name="label_temperature">
     <property name="text">
      <string>Température actuelle: 20°C</string>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSlider_puissance">
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_puissance">
     <property name="text">
      <string>Puissance: 0%</string>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSlider_consigne">
     <property name="maximum">
      <number>65</number>
     </property>
     <property name="minimum">
      <number>20</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_consigne">
     <property name="text">
      <string>Consigne: 20°C</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_start">
     <property name="text">
      <string>Démarrer la chauffe</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_stop">
     <property name="text">
      <string>Arrêter la chauffe</string>
     </property>
    </widget>
    <widget class="QVBoxLayout" name="verticalLayout_graphic">
    </widget>
   </layout>
  </widget>
 </widget>
 <connections/>
</ui>
